recipe/meta.yaml

```diff
@@ -28,17 +28,21 @@ requirements:
     - argh >=0.24.1
     - pathtools >=0.1.1
     - pyyaml >=3.10
+    - pyobjc-framework-cocoa >=4.2.2  # [osx]
+    - pyobjc-framework-fsevents >=4.2.2  # [osx]
 
 test:
+  requires:
+    - pip
   imports:
     - watchdog
     - watchdog.observers
+    - watchdog.observers.fsevents  # [osx]
     - watchdog.tricks
     - watchdog.utils
   commands:
+    - python -m pip check
     - watchmedo --help
-    - test -f $SP_DIR/_watchdog_fsevents.so  # [osx and py2k]
-    - test -f $SP_DIR/_watchdog_fsevents.cpython-37m-darwin.so  # [osx and py==37]
```

Comment:
Why did you remove these tests?

Reply:
I did replace them with an import test (forgot to push) and added a C compiler as a requirement as this is used during the build of them.

Reply:
I do not think pip check replaces these checks

Reply:
@goanpeca No, they are replaced by an import check as above. These are Python modules where the so-name depends on the Python version, the Python module name is though version independent.